operations/network/management_network/canu_install_update.md

```diff
@@ -0,0 +1,39 @@
+  # Upgrade Canu
+
+Before using CANU (CSM Automated Network Utility) to test, validate or configure your network. Please ensure that Canu is running on the latest version. 
+
+   * If Canu is already installed in your system, you can check the canu version with following command: 
+
+```text
+canu --version
+```
+
+Upgrade/Install procedure:
+
+   * Download latest CANU from link below.  
+
+   * [Canu releases](https://github.com/Cray-HPE/canu/releases)
+
+   * Once you have succesfully moved the new RPM to your system, wether that is your workstation or live system. You can issue the following command(s) to upgrade or install Canu:   
```

Comment:
```suggestion
1. After the new RPM is on either the workstation or live system, wether that is your workstation or live system, issue the following command(s) to upgrade or install CANU.
```

Reply:
I added some context above so we can simplify this language with my suggestion below.